dvc/config.py

```diff
@@ -468,6 +469,11 @@ def validate(data):
             raise ConfigError(str(exc)) from None
 
 
+def _format_config(config):
+    for key, value in flatten(config).items():
+        yield f"{key}={value}"
```

Comment:
Let's move this to the `dvc/command/config.py`, since it is only used there.

We might reconsider the move in the future if we decide to implement `--show-origin`, but for now better to just keep it closer to where it is used :slightly_smiling_face:

Reply:
The test you have for it, could be moved to `tests/unit/command/test_config.py`. Just for consistency, we try to keep tests for such helpers there.

Reply:
Though we don't have test_config.py there yet, so I'm fine with keeping the tests where they are right now :ok_hand: